### ISSUE TITLE
Give a convenient way to deploy chaos-dashboard with UI on local

### ISSUE THREAD

**Issue by Colstuwjx** [chaos-mesh/chaos-mesh#1083]
## Feature Request

**Is your feature request related to a problem? Please describe:**

From [this part](https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md#step-3-start-chaos-mesh-locally-and-do-manual-tests), we could get a dashboard image via `make image`, and we could deploy it on kind cluster. However, we didn't deploy the dashboard UI...

how to make it via makefile? `UI=1 make image`? I've tried, but I still need to setup port-forward to see the ui:

```
kubectl port-forward --address 0.0.0.0 svc/chaos-dashboard 2333:2333 -n chaos-testing
```

even did this, I still could see some error logs(about no table events) at each build:
```
2020-10-17T11:09:35.206Z	ERROR	experimentStore	failed to delete all incomplete experiments	{"error": "no such table: events"}
github.com/go-logr/zapr.(*zapLogger).Error
	/root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/pkg/store/experiment.NewStore
	/src/pkg/store/experiment/experiment.go:38
reflect.Value.call
	/usr/local/go/src/reflect/value.go:460
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:321
go.uber.org/dig.(*node).Call
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/dig.go:667

go.uber.org/dig.paramSingle.Build
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:245
(/src/pkg/store/experiment/experiment.go:51)
go.uber.org/dig.paramList.BuildList
[2020-10-17 11:09:35]  no such table: events
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:201
go.uber.org/dig.(*node).Call
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/dig.go:658
go.uber.org/dig.paramSingle.Build
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:245
go.uber.org/dig.paramList.BuildList
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:201
go.uber.org/dig.(*node).Call
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/dig.go:658
go.uber.org/dig.paramSingle.Build
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:245
go.uber.org/dig.paramList.BuildList
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/param.go:201
go.uber.org/dig.(*Container).Invoke
	/root/go/pkg/mod/go.uber.org/dig@v1.9.0/dig.go:389
go.uber.org/fx.(*App).executeInvokes
	/root/go/pkg/mod/go.uber.org/fx@v1.12.0/app.go:659
go.uber.org/fx.New
	/root/go/pkg/mod/go.uber.org/fx@v1.12.0/app.go:438
main.main
	/src/cmd/chaos-dashboard/main.go:80
runtime.main
	/usr/local/go/src/runtime/proc.go:203
2020-10-17T11:09:35.548Z	INFO	controller-runtime.metrics	metrics server is starting to listen	{"addr": ":8080"}
2020/10/17 11:09:35 [Fx] INVOKE		github.com/chaos-mesh/chaos-mesh/pkg/apiserver/experiment.Register()
2020/10/17 11:09:35 [Fx] INVOKE		github.com/chaos-mesh/chaos-mesh/pkg/apiserver/event.Register()
[GIN-debug] POST   /api/common/pods          --> github.com/chaos-mesh/chaos-mesh/pkg/apiserver/common.(*Service).listPods-fm (4 handlers)
[GIN-debug] GET    /api/common/namespaces    --> github.com/chaos-mesh/chaos-mesh/pkg/apiserver/common.(*Service).getNamespaces-fm (4 handlers)
```

As a result, we should offer a robust and convenient way to deploy chaos-dashboard with UI on local.

**Describe the feature you'd like:**

Deploy chaos dashboard with UI, full worked on local.

**Describe alternatives you've considered:**

None.

**Teachability, Documentation, Adoption, Migration Strategy:**

Should update CONTRIBUTING.md.

**Comment by mikechengwei** [comment]
We can add port-forward dashboard ui when PR #1026 merged.

**Comment by g1eny0ung** [comment]
@Colstuwjx For now you can run `UI=1 make chaos-dashboard` and then exec it with `./bin/chaos-dashboard`.

> 2020-10-17T11:09:35.206Z	ERROR	experimentStore	failed to delete all incomplete experiments	{"error": "no such table: events"}

And also, this error has already fixed in #1029 (https://github.com/chaos-mesh/chaos-mesh/pull/1029/files#diff-fb9f8e8a65e63ef4cf1e15404b8bc0aa9d3694209d9ba484984f6a78f2207b64R29).

We will speed up the review process to fix this problem.
